Explorer/Assets/DCL/AvatarRendering/AvatarShape/UnityInterface/AvatarBase.cs

```diff
@@ -8,11 +8,13 @@ namespace DCL.AvatarRendering.AvatarShape.UnityInterface
 {
     public class AvatarBase : MonoBehaviour, IAvatarView
     {
-        [SerializeField] private Animator avatarAnimator;
         private List<KeyValuePair<AnimationClip, AnimationClip>> animationOverrides;
         private AnimationClip lastEmote;
 
         private AnimatorOverrideController overrideController;
+
```

Comment:
`AvatarAnimationEventHandler ` component is located on the `gameObject `with `AvatarAnimator`, because animation events callbacks are called implicitly and script should be on the same gameObject for that.  Unfortunatly, I wasn't able to explicitly expose `IAvatarAnimationEventHandler` due to cyclic dependecies, so I went not that clean way with `AvatarAnimator.gameObject.GetComponent<AvatarAnimationEventHandler>`  in my analytics class